homeassistant/components/ovo_energy/config_flow.py

```diff
@@ -30,7 +35,9 @@ async def async_step_user(self, user_input=None):
             client = OVOEnergy()
             try:
                 authenticated = await client.authenticate(
-                    user_input[CONF_USERNAME], user_input[CONF_PASSWORD]
+                    user_input[CONF_USERNAME],
+                    user_input[CONF_PASSWORD],
+                    user_input.get(CONF_ACCOUNT, None),
```

Comment:
`None` is the default value if the key is missing in the dict, when using `dict.get`.